packages/validation-manager/src/GethTracer.ts

```diff
@@ -35,7 +35,7 @@ export async function debug_traceRip7560Validation (provider: JsonRpcProvider, t
   const tx1 = await resolveProperties(tx)
   return await provider.send('debug_traceRip7560Validation', [tx1, 'latest']).catch(e => {
     debug('ex=', e.error)
-    throw e
+    throw new RpcError(e.error, 0)
```

Comment:
'code' is optional for `RpcError`, but code `0` is not defined anywhere, so why bother?